pyvista/core/pyvista_ndarray.py

```diff
@@ -0,0 +1,111 @@
+"""Contains pyvista_ndarray a numpy ndarray type used in pyvista."""
+
+import numpy
+from pyvista.utilities.helpers import convert_array, FieldAssociation
+from vtk.numpy_interface.dataset_adapter import VTKObjectWrapper, VTKArray
+from vtk.vtkCommonCore import vtkWeakReference
+from vtk.vtkCommonKitPython import vtkDataArray, vtkAbstractArray
+
+
+class pyvista_ndarray(VTKArray):
+    """
+    A type of numpy.ndarray which stores a reference to a vtk array as well as the owning dataset.
```

Comment:
I tried to wrap but couldn't get pydoc to accept it because the first line is a summary and has to be one line. I can reword the docstring to be shorter.

Reply:
I've changed all the docstrings in pyvista_ndarray to be under or very close to 72.